lib/publiccloud/ec2.pm

```diff
@@ -241,6 +241,23 @@ sub stop_instance
     die("Failed to stop instance $instance_id") unless ($attempts > 0);
 }
 
+sub hibernate_instance {
```

Comment:
Since hibernation is not requested or covered or supported, I would suggest to remove those methods.